config/routes.rb

```diff
@@ -148,6 +148,7 @@
     resources :user_creations, only: [:destroy] do
       member do
         get :hide
+        get :set_spam
```

Comment:
Can we make this a put request? If it's updating data, it really shouldn't be 'get'.